CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## \[0.35.0]
+
+- [`dbea0f3`](https://github.com/tauri-apps/wry/commit/dbea0f3b1ff15dbb70902190336ba38b01a0e0f8)([#1071](https://github.com/tauri-apps/wry/pull/1071)) **Breaking change** Consistently use `WebView` in API names. The following APIs were renamed:
+
+  - `WebviewExtWindows` → `WebViewExtWindows`
+  - `WebviewExtUnix` → `WebViewExtUnix`
+  - `WebviewExtMacOS` → `WebViewExtMacOS`
+  - `WebviewExtIOS` → `WebViewExtIOS`
+  - `WebviewExtAndroid` → `WebViewExtAndroid`
+  - `WebviewUriLoader` → `WebViewUriLoader`
+- [`dbea0f3`](https://github.com/tauri-apps/wry/commit/dbea0f3b1ff15dbb70902190336ba38b01a0e0f8)([#1071](https://github.com/tauri-apps/wry/pull/1071)) Add `WebViewExtWindows::set_memory_usage_level` API to set the [memory usage target level](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2memoryusagetargetlevel) on Windows. Setting 'Low' memory usage target level when an application is going to inactive can significantly reduce the memory consumption. Please read the [guide for WebView2](https://github.com/MicrosoftEdge/WebView2Feedback/blob/main/specs/MemoryUsageTargetLevel.md) for more details.
+- [`dbea0f3`](https://github.com/tauri-apps/wry/commit/dbea0f3b1ff15dbb70902190336ba38b01a0e0f8)([#1071](https://github.com/tauri-apps/wry/pull/1071)) Refactor new method to take raw window handle instead. Following are APIs got affected:
```

Comment:
These PR links are wrong because I edited the change files at #1071. Shall I create a PR to this PR branch for fixing those links?

Reply:
Just add a review suggestion and I can update. Or I can still do this when it's about to publish

Reply:
I added suggestions.